components/admin_real/desktop/ReservationFieldsModal.jsx

```diff
@@ -1,17 +1,36 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import Modal from "antd/lib/modal/Modal";
-import { Alert, DatePicker, Input } from "antd";
+import { Form, Button, DatePicker, Input, InputNumber, Radio } from "antd";
 import { TimePicker } from "antd";
 import moment from "moment";
-import { Select } from "antd";
 import { dateFormat, timeFormat } from "../shared/constants";
-import FormField from "./FormField";
 import { useUser } from "../../../utils/useUser";
-import { openErrorNotification } from "../shared/reservationTableUtils";
-import { openSuccessNotification } from "../shared/reservationTableUtils";
+import {
+  NUM_OF_DIVES,
+  DEEPEST_DIVE,
+  DEEPEST_TO_TEXT_MAPPING,
+  NUM_OF_DIVES_TO_TEXT_MAPPING,
+} from "../../../utils/supabase";
 import { TableContext } from "../../states/ReservationTableContextProvider";
+import { openSuccessNotification } from "../shared/reservationTableUtils";
 
-const { Option } = Select;
+const formatValues = (values) => {
```

Comment:
Instead of formatting on the API call, I moved the logic here

Reply:
👍